emails/_footer.liquid

```diff
@@ -1,29 +1,32 @@
-<center>
+<center class="footer u-padding-vertical-lg u-margin-top-lg">
```

Comment:
Should never use margin in emails, hotmail/gmail strip margin out, so always use padding.